app/Http/Controllers/ApplicationTimelineController.php

```diff
@@ -12,6 +12,7 @@ public function show(JobApplication $jobApplication)
         return view('applicant/application-timeline-root')
             ->with([
                 'title' => $jobApplication->job_poster->title, // TODO: Check with design what the title should be.
+                'diable_clone_js' => true,
```

Comment:
This is typo'd, so it shouldn't actually have fixed anything 🤔

Reply:
It seems it was an intermittent issue in the first place, and I just got lucky after adding this typo'd line 😅 . Fixing the typo _definitely_ helped.